### ISSUE TITLE
Dialog tab order

### ISSUE THREAD

**Issue by agilitylab** [radzenhq/radzen-blazor#1585]
You can tab from "confirm" to "cancel" button in dialogs but cannot tab through "close" button. Also, tab order doesn't wrap e.g. when you get to the end of the tab sequence, it stops tabbing although you can go back the way using Shift+Tab.

These have been flagged by our QA team as accessibility issues.

Steps to reproduce the behaviour:

1. Go to [Radzen Confirm Dialog](https://blazor.radzen.com/dialog#confirm-dialog)
2. Click on "Confirm dialog" button
3. Try tabbing
4. Observe behaviour

Expected behaviour is being able to cycle focus through close, confirm and cancel buttons in the dialog using the tab button (as is the case with blazorise modal - see [Blazorise Modal component](https://blazorise.com/docs/components/modal)

**Comment by enchev** [comment]
I'll push update on the focus logic to allow cycle between first and last elements in the dialog content however this will not include the elements in the title (the close button is there), still I've enabled ENTER and SPACE keys to be used to close the dialog if pressed over close button. You can close the dialog using ESC key.

**Comment by agilitylab** [comment]
Thank you!

**Comment by agilitylab** [comment]
Any future plans to enable cycling through the close button in the title as well as the dialog content elements?

**Comment by enchev** [comment]
We will try to enable this however we are not sure if we will be able.
